app/Validation/PasswordEmailValidation.php

```diff
@@ -36,4 +36,14 @@ public function isNotSuperAdmin($admin)
 
         return true;
     }
+
+    public function validateConfirmedAccount($user)
+    {
+        if(!$user || $user["confirmed"] == 0) {
+            throw new Exception("Tu cuenta no ha terminado de ser confirmada. Por favor, revisa tu correo electrónico para finalizar el proceso de confirmación de tu cuenta.");
```

Comment:
I think the message here should be the same as the login one, it sounds better `Tu cuenta no ha sido confirmada` instead of `Tu cuenta no ha terminado de ser confirmada` because it makes me think I already started the confirmation process when I really haven't done the confirmation.
```suggestion
            throw new Exception("Tu cuenta no ha sido confirmada. Por favor, revisa tu correo electrónico para finalizar el proceso de confirmación de tu cuenta.");
```